src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandParser.cs

```diff
@@ -51,9 +45,16 @@ internal static string GetWorkloadsVersion(WorkloadInfoHelper workloadInfoHelper
         internal static void ShowWorkloadsInfo(ParseResult parseResult = null, WorkloadInfoHelper workloadInfoHelper = null, IReporter reporter = null, string dotnetDir = null, bool showVersion = true)
         {
             workloadInfoHelper ??= new WorkloadInfoHelper(parseResult != null ? parseResult.HasOption(SharedOptions.InteractiveOption) : false);
+            reporter ??= Utils.Reporter.Output;
+            var globalJsonInformation = workloadInfoHelper.ManifestProvider.GetGlobalJsonInformation();
+            if (globalJsonInformation?.WorkloadVersionInstalled == false)
```

Comment:
probably don't need the `== false`,

Reply:
globalJsonInformation could be null, and if not, WorkloadVersionInstalled could be true or false, so this is really a bool?, hence the == false.